### PR TITLE
Improve CMS README, documentation, and instructions

### PR DESCRIPTION
**Pull request by ccostino**
Part of #609 
Part of #608
Part of #607 

This changeset represents the beginning of updates to our local development workflow, including updates to our documentation.  As more work is completed this will continue to be updated to account for new/updated commands and information.

This PR currently accounts for all of the documentation updates to correspond with #620.  Going forward we'll incorporate the documentation changes directly with the other PRs.

/cc @jontours, @LindsayYoung, @xtine, @emileighoutlaw

### CODE REVIEW

**Comment by jontours** [comment]
@ccostino I just ran the cms locally for the first time and found a slight discrepancy for the readme.  In migration steps it is stated to run the various tasks in `manage.py`.  However, at the root directory level for `fec-cms`, manage.py is located under the `fec` directory.  Maybe just change the commands in the documents to `fec\manage.py taskXYZ`.

**Comment by jontours** [comment]
Oh wait, that change is in this PR!  Or you are just very fast!

**Comment by ccostino** [comment]
*blow smoke off fingers* :-)

Yeah I noticed that during my first pass, it'll be addressed here!  Thanks, @jontours!

**Comment by ccostino** [comment]
Wow, something got botched here, I must've accidentally merged or committed some other work from one of my other branches.  I'm force pushing an update here to bring in just the README updates I've been working on.

**Comment by ccostino** [comment]
@emileighoutlaw, I tagged you on this as well just to make sure from a content perspective things still look good compared to the other improvements we had made to other documentation!  If you have a moment and any additional feedback or questions please don't hesitate to share them. :-)

File: README.md [hunk 1]
```diff
@@ -1,144 +1,249 @@
...
+```bash
+cd fec/
+./manage.py runserver
+```
 
-    FEC_APP_URL = 'http://localhost:3000'
+## Enabling/toggling features
+[settings.py](https://github.com/18F/fec-cms/blob/develop/fec/fec/settings/base.py)
+includes a set of `FEATURES` which can be enabled using environment flags or
+via `settings.local`, e.g.:
```
Comment (emileighoutlaw) [inline README.md]: @ccostino — should there be something here after the "e.g.:" ?

**Comment by emileighoutlaw** [comment]
Committed a couple punctuation edits and added on question. Otherwise, this looks great! Thanks for tagging me, Carlo!

File: README.md (hunk 1 above)
Comment (ccostino) [inline README.md]: Ah good catch!  It's not entirely clear here; this was a section I just adjusted spacing on, but I can amend it a bit to help fix it up. :-)

**Comment by ccostino** [comment]
You're very welcome, thank you for taking the time and going through the changes! :-)

**Pull request merged**